test/files/run/repl-classbased.check

```diff
@@ -6,18 +6,26 @@ scala> class C { implicit val k: K = K("OK?"); override def toString = s"C($k)"
 defined class C
 
 scala> val c = new C
-c: C = C(K(OK?))
+c: INSTANCE.C = C(K(OK?))
```

Comment:
I intended very little with my changes beyond the impetus to get this lacrosse the line.